core/domain/platform_feature_services_test.py

```diff
@@ -159,13 +184,36 @@ def test_get_all_feature_flag_values_in_dev_returns_correct_values(
                     context),
                 {
                     self.dev_feature.name: True,
+                    self.test_feature.name: True,
+                    self.prod_feature.name: True,
+                })
+
+    def test_get_all_feature_flag_values_in_test_returns_correct_values(
+        self
+    ) -> None:
+        constants_swap = self.swap(constants, 'DEV_MODE', False)
+        project_id_swap = self.swap(feconf, 'OPPIA_PROJECT_ID', 'testserver')
+        with constants_swap, project_id_swap:
+            context = feature_services.create_evaluation_context_for_client({
+                'platform_type': 'Android',
+                'browser_type': None,
+                'app_version': '1.0.0',
+            })
+            self.assertEqual(
+                feature_services.evaluate_all_feature_flag_values_for_client(
+                    context),
+                {
+                    self.dev_feature.name: False,
+                    self.test_feature.name: True,
                     self.prod_feature.name: True,
                 })
 
     def test_get_all_feature_flag_values_in_prod_returns_correct_values(
         self
     ) -> None:
-        with self.swap(constants, 'DEV_MODE', False):
+        constants_swap = self.swap(constants, 'DEV_MODE', False)
+        project_id_swap = self.swap(feconf, 'OPPIA_PROJECT_ID', 'oppiaserver')
```

Comment:
Nit: These can be set in `setUp` on `self` and then used throughout the test functions.

Reply:
Ah thanks. But these are kind of different for different tests since we set them to different things?

Reply:
Yeah looking at it I think it is fine as there is not too much of them